packages/bpk-component-loading-button/src/BpkLoadingButton.tsx

```diff
@@ -156,39 +156,4 @@ const BpkLoadingButton = (props: LoadingProps) => {
   );
 };
 
-BpkLoadingButton.propTypes = {
-  children: PropTypes.node.isRequired,
-  className: PropTypes.string,
```

Comment:
Should `classname` be included in the destructed props above?

Reply:
Thats a good catch! Having a deeper look, its not actually used in the component itself! So I am wondering if this was added for typing in the past but would just follow `...rest` and go down to the BpkButton.

AFAIK flow/propTypes didn't have the concept of merging props with that of other components 🤔